index.js

```diff
@@ -1,2 +1,5 @@
+const Dist = require('./dist')
+
 // This file is mainly just for cleaning up the exports to be intuitive/commonjs
-module.exports = require('./dist')
+module.exports = Dist.default
+module.exports.default = Dist.default
```

Comment:
This preserves the original behavior...and maybe we can rip it out in the next major release?

Or rip it out now and do a major? Or rip it out now and do a minor/patch and semi-break semver?